package.json

```diff
@@ -101,7 +101,7 @@
     "screwdriver-models": "^27.0.0",
```

Comment:
should pull in latest models too after https://github.com/screwdriver-cd/models/pull/246 is merged

Reply:
Should be ^27.2.2